docs/02.API-REFERENCE.md

```diff
@@ -3787,11 +3787,13 @@ is no longer needed.
 jerry_value_t
 jerry_exec_snapshot (const uint32_t *snapshot_p,
                      size_t snapshot_size,
+                     size_t func_index,
```

Comment:
Do you think having multiple snapshots in one will be common?
If not, I think there's value in having a simpler API w/o the index.